packages/sdks/snippets/react/src/components/CustomHero.tsx

```diff
@@ -0,0 +1,11 @@
+import { ReactNode } from 'react';
+
+interface CustomHeroProps {
+  children: ReactNode;
+}
+
+const CustomHero: React.FC<CustomHeroProps> = (props) => {
+  return <div className="Hero">{props.children}</div>;
```

Comment:
I would add some style or text in this component here so that it's not an empty plain container of Builder content